packages/nextjs/src/common/utils/addHeadersAsAttributes.ts

```diff
@@ -15,6 +15,10 @@ export function addHeadersAsAttributes(
   const client = getClient();
   const sendDefaultPii = client?.getOptions().sendDefaultPii ?? false;
 
+  if (!sendDefaultPii) {
+    return {};
+  }
```

Comment:
We still send the headers as attributes when this is `false`, so you can delete this statement. Only sensitive headers are not sent with that.